src/main/java/frc/robot/constants/Constants.java

```diff
@@ -61,4 +61,14 @@ public static class Shooter {
             Units.inchesToMeters(5.727224),
             new Rotation3d(0.0, 0.0, Math.toRadians(180.0)));
   }
+
+  public static class Intake {
+    public static final Rotation2d MIN_ANGLE = new Rotation2d(); // TODO
+    public static final Rotation2d MAX_ANGLE = new Rotation2d(); // TODO
+
+    public static final Rotation2d GROUND_INTAKE_ANGLE = new Rotation2d(); // TODO
+    public static final Rotation2d STOW_ANGLE = new Rotation2d(); // TODO
```

Comment:
Determine these values empirically or else the intake won't work and will fling itself into the bumpers or indexer. Robot shouldn't be enabled as is.